src/Functional.CQS.AOP.IoC.PureDI.MetricsCapturing/Functional.CQS.AOP.IoC.PureDI.MetricsCapturing.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <DocumentationFile>D:\Projects\Functional.CQS\src\Functional.CQS.AOP.IoC.PureDI.MetricsCapturing\Functional.CQS.AOP.IoC.PureDI.MetricsCapturing.xml</DocumentationFile>
```

Comment:
In both `Debug` and `Release` configurations, machine-specific paths should be replaced with relative paths.